data-es6.js

```diff
@@ -3570,8 +3591,9 @@ exports.tests = [
         var obj = { foo: 1, bar: 2 };
         var iterator = Reflect.enumerate(obj);
 
+        var passed = Symbol.iterator in item;
```

Comment:
"All modern browsers"? You neglect Safari, iOS, and IE 11 (which is still the latest stable version of that browser line).